test/setup.js

```diff
@@ -7,6 +7,6 @@ const truncateTable = model =>
 
 const truncateDatabase = () => Promise.all(tables.map(truncateTable));
 
-global.beforeEach(async () => {
+global.beforeAll(async () => {
```

Comment:
Why is this change?

Reply:
I change it to persist the user data and then test that the email already exists in the database

Reply:
I restore it and modify the specific test